### ISSUE TITLE
force-level1-csv area of interest

### ISSUE THREAD

**Issue by jakimowb** [davidfrantz/force#40]
force-level1-csd fails to read the area of interest from ascii testfiles and command line, at least for the following polygon

projectaoi.txt
````
-57.11,-10.13
-57.11,-5.42
-54.17,-5.42
-54.17,-10.13
-57.11,-10.13
````
Error:
````
Tile list as AOI detected.

Error: One or more tiles seem to be formatted incorrectly.
Please check -57.11.-10.13
````

The following command was used on BB8:
````
force-level1-csd \
/data/Jakku/diss_bj/level1 \
/data/Jakku/diss_bj/level1 \
/data/Jakku/diss_bj/level1/queue.txt \
/data/Jakku/diss_bj/level1/projectaoi.txt
````

**Setup**
- FORCE 3.5.2
- Ubuntu 18 (BB8)

**Comment by davidfrantz** [comment]
Dear Benjamin,

this is due to an error in usage, quoting from the help:

``` bash
 area of interest
  (1) user-supplied coordinates of your study area:
      The polygon must be closed (first X/Y = last X/Y). X/Y must be given as
      decimal degrees with negative values for West and South coordinates.
      You can either give the path to a file, or give the coordinates on the command line.
      If in the file, put one coordinate per line.
      If on the command line, give a comma separated list.
```

X and Y are separated with ``/``, not ``,``.

Cheers,
David

**Comment by ernstste** [comment]
Thanks for reporting Benjamin.
David is correct, the lat lon coordinate pairs need to be separated with a forward dash `/`, just like in the 'old' force-level1-sentinel2 module. 
However, there are a few more things going on here. Negative values can break the CLI parsing and the OWS server does not seem to like the request that was working fine when the tool was tested before. Was anything changed there?

**Comment by jakimowb** [comment]
Update: I fixed the coordinate X/Y separator to
````
cat projectaoi.txt
-57.11/-10.13
-57.11/-5.42
-54.17/-5.42
-54.17/-10.13
-57.11/-10.13

````

but still get the following error:

````
Incorrect number of mandatory input arguments provided
Expected: 4 Received: 8: --keep-meta,--daterange,20200101,20201231,--,/data/Jakku/diss_bj/level1,/data/Jakku/diss_bj/level1,/data/Jakku/diss_bj/level1/queue.txt,/data/Jakku/diss_bj/level1/projectaoi.txt
````

Removing all optional arguments returns 

````
 force-level1-csd \
level1 \
level1 \
level1/queue.txt \
level1/projectaoi.txt

grep: level1/projectaoi.txt: No such file or directory
Error: At least one of the AOI coordinates does not seem to be in the format LAT/LON
Coordinate: level1/projectaoi.txt

````

**Comment by ernstste** [comment]
Hi Benjamin,
as mentioned before, there are a few more things going on here. I fixed all of them earlier today, but the issue with the server remains and I'm not sure what the problem is.

**Comment by ernstste** [comment]
This has been fixed in pull request #42 and will be implemented with the next release.
